Apps/Playground/scripts/postinstall.js

```diff
@@ -0,0 +1,6 @@
+const shelljs = require('shelljs');
+const version = shelljs.exec('npm --version');
+
+if (!version.trim().match(/6\.\d+\.\d+/g)) {

```

Comment:
Sorry I know there has already been some discussion on this and I meant to comment on it earlier but forgot... you could also consider using the `semver` package, which would make this code a bit more readable. Something like:
```js
const semver = require('semver');
const shelljs = require('shelljs');
const version = semver.clean(shelljs.exec('npm --version'));

if (!semver.lt(version, '7')) {
    throw ...
}
```
Or you might need to do `semver.lt(version, semver.coerce('7'))`, not sure. Anyway up to you, but the `semver` package is all about dealing with versions and version comparisons.

Reply:
sounds good. going to go with the regex for now, we can update this in the future as needed!